src/osx/carbon/graphics.cpp

```diff
@@ -2333,10 +2322,9 @@ void wxMacCoreGraphicsContext::GetTextExtent( const wxString &str, wxDouble *wid
     {
         if ( width )
             *width = w;
-        if ( height )
-            *height = a+d+l;
     }
-
+    if ( height )
```

Comment:
oh, sorry, I thought I had made a mistake when writing it. I had expected it to still return the height of the font rectangle and stumbled upon it when using it, it seemed so obvious to be a bug on my part that I fixed without even checking the other port